### PR TITLE
Add documentation of the core concepts of Jayvee

### PR DESCRIPTION
**Pull request by georg-schwarz**
Partially addresses #98.

Feel free to add improvements directly to the PR in case you think they don't need discussion. @rhazn @felix-oq 

## Impressions
image
image
image

### CODE REVIEW

File: apps/docs/docs/user/intro.md [hunk 1]
```diff
@@ -4,4 +4,8 @@ sidebar_position: 1
 
 # Introduction to Jayvee
 
-tbd
+Jayvee is a domain-specific language (DSL) data engineering - the cleaning and preprocessing of data for later activities like data science or machine learning. You can use Jayvee to **model an ETL pipeline** and the command-line interpreter to **run the ETL pipeline** on your local machine. 
+
+## Getting Started
+
+TBD: installation instructions for interpreter
```
Comment (rhazn) [inline apps/docs/docs/user/intro.md:11]: Can then also be removed from README.md, should be added from npm.

File: apps/docs/docs/user/core-concepts.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+The general structure of a `Pipeline` consisting of different blocks is the following:
+```mermaid
+flowchart LR
+    A[ExtractorBlock] --> B(TransformatorBlock)
+    B --> C(TransformatorBlock)
+    C --> D(LoaderBlock)
+```
+
+The common syntax of blocks is at its core a key-value map to provide configuration to the block. The availability of attribute keys and their respective `ValueTypes` is determined by the type of the `Block` - indicated by the identifier after the keyword `oftype`:
```
Comment (rhazn) [inline apps/docs/docs/user/core-concepts.md:63]: Felix renamed attributes to properties so renaming all references to attributes to properties would make sense.

**Review by rhazn — APPROVED** [review]

File: apps/docs/docs/user/core-concepts.md (hunk 2 above)
Comment (georg-schwarz) [reply apps/docs/docs/user/core-concepts.md]: Addressed in [b809763](https://github.com/jvalue/jayvee/pull/244/commits/b809763ac8ea5296b286f6ac862ab440e604632b)

**Comment by georg-schwarz** [comment]
I will merge this PR. If you catch some more issues be sure to open another PR ;-)

**Pull request merged**